test/e2e/network/dns_common.go

```diff
@@ -110,6 +110,9 @@ func (t *dnsTestCommon) runDig(dnsName, target string) []string {
 	default:
 		panic(fmt.Errorf("invalid target: " + target))
 	}
+	if strings.HasSuffix(dnsName, "in-addr.arpa") {
```

Comment:
minor: what if there is a trailing dot (e.g. `XXX.in-addr.arpa.`) ?

Reply:
Done.